src/fuse_ctrl/data/otp_ctrl_mmap.hjson

```diff
@@ -114,153 +266,200 @@
-                    size:   "48"
-                }
+                    name: "MANUF_TOKEN",
+                    inv_default: "<random>",
+                    size: "16"
+                },
             ],
-            desc:   '''Secret partition 1.
-            This contains Obfuscated UDS seed. 
+            desc: '''Secret life-cycle manufacturing token partition.
             '''
-        }
+        },
         {
-            name:       "SECRET2",
+            name:       "SECRET_LC_PROD_PARTITION",
             variant:    "Buffered",
             secret:     true,
             sw_digest:  false,
             hw_digest:  true,
             write_lock: "Digest",
             read_lock:  "Digest",
-            key_sel:    "Secret2Key",
+            key_sel:    "SecretLifeCycleProdKey",
             integrity:  true,
             bkout_type: false,
             items: [
                 {
-                    name:   "FIELD_ENTROPY",
-                    size:   "32"
-                }
+                    name: "PROD_TOKEN",
+                    inv_default: "<random>",
+                    size: "16"
+                },
             ],
-            desc: '''Secret partition 2. 
-            This contains obfuscated field entropy. 
+            desc: '''Secret life-cycle production token partition.
             '''
-        }
+        },
         {
-            name:       "SECRET3",
+            name:       "SECRET_LC_RMA_PARTITION",
             variant:    "Buffered",
             secret:     true,
             sw_digest:  false,
             hw_digest:  true,
             write_lock: "Digest",
             read_lock:  "Digest",
-            key_sel:    "Secret3Key",
+            key_sel:    "SecretLifeCycleRMAKey",
             integrity:  true,
             bkout_type: false,
             items: [
                 {
-                    name:   "KEY_MANIFEST_PK_HASH",
-                    size:   "48"
-                }
-                { 
-                    name:   "RMA_TOKEN",
+                    name: "RMA_TOKEN",
                     inv_default: "<random>",
-                    size:   "16"
+                    size: "16"
+                },
+            ],
+            desc: '''Secret life-cycle RMA token partition.
+            '''
+        },
+        {
+            name:       "SVN_PARTITION",
+            variant:    "Unbuffered",
+            absorb:     false,
+            secret:     false,
+            sw_digest:  false,
+            hw_digest:  false,
+            write_lock: "None",
+            read_lock:  "CSR",
+            key_sel:    "NoKey",
+            // This is a strike counter, hence we need to disable ECC integrity for this to work.
+            // Integrity is handled at a higher level by SW as described below.
+            integrity:  false,
+            bkout_type: false,
+            items: [
+                {
+                    name: "FMC_KEY_MANIFEST_SVN",
```

Comment:
Multi-bit encoding for redundancy is recommended. So 32-bits may result in 16 SVN versions. We should check if this is sufficient.